packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -350,6 +350,31 @@ mixin RendererBinding on BindingBase, ServicesBinding, SchedulerBinding, Gesture
     return ViewConfiguration.fromView(renderView.flutterView);
   }
 
+  /// Create a [SceneBuilder].
+  ///
+  /// This hook enables test bindings to instrument the rendering layer.
+  ///
+  /// This is used by the [RenderView] to create the [SceneBuilder] that is
+  /// passed to the [Layer] system to render the scene.
+  ui.SceneBuilder createSceneBuilder() => ui.SceneBuilder();
```

Comment:
I don't feel very strongly about this. It's an API that few people will ever use, and those who use it will presumably want to use correctly, or will have very good reasons for doing weird things with it. If you would like me to annotate it though, let me know. :-)

Reply:
No strong feelings on tis one either way.